packages/vite/src/node/plugins/resolve.ts

```diff
@@ -240,6 +240,12 @@ function tryFsResolve(
   }
 
   let res: string | undefined
+  if (
+    (res = tryResolveFile(file, postfix, options, false, options.tryPrefix))
```

Comment:
**thought (dx, readability):** Is this the same as  
```ts
const res: string | undefined = tryResolveFile(file, postfix, options, false, options.tryPrefix)
if (res) return res
```

If so, I think we should write this this way, cause it's much more readable

Reply:
You mean:
```JS
  let res: string | undefined
  res = tryResolveFile(file, postfix, options, false, options.tryPrefix)
  if (res) return res

  for (const ext of options.extensions || DEFAULT_EXTENSIONS) {
    res = tryResolveFile(
      file + ext,
      postfix,
      options,
      false,
      options.tryPrefix
    )
    if (res) return res
  }
  res = tryResolveFile(file, postfix, options, tryIndex, options.tryPrefix)
  if (res) return res
```

Reply:
I agree, it looks better. I just tried to touch as little code as possible and maintain original style

Reply:
Yeah :sweat_smile:, I haven't scrolled down the code so much in the GitHub Web GUI
But yes, please try to improve the code and don't fear touching it :slightly_smiling_face:

Reply:
It is not bad to refactor code, but let's try to follow Evan's style if there is not a clear gain 👍🏼 
It is easier to check the PRs that way